File: 10月份/OC本地gif图/图片浏览器/FCCPicture.m
//
//  FCCPicture.m
//  图片浏览器
//
//  Created by codygao on 16/10/9.
//  Copyright © 2016年 HM. All rights reserved.
//

#import "FCCPicture.h"

@implementation FCCPicture

-(instancetype)initWithDic:(NSDictionary*)dic{
    
    if (self = [super init]){
        [self setValuesForKeysWithDictionary:dic];
    }
    return  self;
}


+(instancetype)pictureWithDic:(NSDictionary*)dic{
    
    return [[self alloc] initWithDic:dic];
    
}

-(void)setValue:(id)value forUndefinedKey:(NSString *)key{
    
}

@end

File: 10月份/10.9/Swift本地gif图/Swift本地gif图/FCCpicture.swift
//
//  FCCpicture.swift
//  Swift本地gif图
//
//  Created by codygao on 16/10/9.
//  Copyright © 2016年 HM. All rights reserved.
//

import UIKit

class FCCpicture: NSObject {

    var icon: String?
    var desc: String?
    var count: NSNumber = 0
    
    init(dic: [String: Any]) {
        super.init()
        setValuesForKeys(dic)
    }
    
    override func setValue(_ value: Any?, forUndefinedKey key: String) {
        
    }
    
}
